install_pim/docker/installation_docker.rst

```diff
@@ -133,7 +133,7 @@ Akeneo PIM is configured with environment variables. The `Symfony documentation
 Running jobs
 ************
 
-Akeneo PIM implements a queue for the `jobs <../../import_and_export_data/index.html>`_, as a PHP daemon. This daemon is a Symfony command, that can only execute one job at a time. It does not consume any other job until the job is finished.
```

Comment:
there's probably an issue left with the link checks, the good links are the `.html` ones, `.rst` does not work

Reply:
Are they? Or maybe this version of Sphinx now expect you to put the name of the .rst file and will convert the link to .html automatically?

Reply:
I just checked, and they're indeed broken. Mathias is right.

Reply:
And with .html the linkchecker is not happy at all, because it cannot find the files... So the problem is on the way we do the link in the code (or the sequence: maybe we should build HTML pages before doing the link check?).

Reply:
Okay this can be solved by using a proper link to a doc reference:
`
To learn more about spin, please refer to :doc:`spin`.
`